sources/packages/web/src/views/institution/locations/programs/LocationProgramOffering.vue

```diff
@@ -98,7 +98,9 @@ export default {
   setup(props: any) {
     const toast = useToastMessage();
     const router = useRouter();
-    const initialData = ref({} as Partial<OfferingFormModel & ProgramDto>);
+    const initialData = ref(
+      {} as Partial<OfferingFormModel & ProgramValidationModel>,
```

Comment:
why do we need to make all `OfferingFormModel` and `ProgramValidationModel` optional?https://www.typescriptlang.org/docs/handbook/utility-types.html

Reply:
Because it incudes create scenario where initialData is only ProgramValidationModel and on update OfferingFormModel  + ProgramValidationModel  .

Reply:
I believe that the Partial is no longer needed.